from utvsapi import exceptions


def preprocessor(cls, function_name, request):
    return
    # This is prepared for the future
    raise exceptions.UnauthorizedException('Bad authentication')
